en/components/categorychart_performance.md

```diff
@@ -6,24 +6,38 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
 ## Performance
```

Comment:
@pamelabrasil could you split the performance doc into two separate documents please? I don't think we should have both samples on the same page, as they will impact each other's performance.